edit_documentation_here/source/installation_guide/installation_guide.md

```diff
@@ -1,184 +1,241 @@
 # Installation
-<br/>
-The developer version of InVesalius runs on Windows, macOS, and Linux operating systems. 
-On this page, instructions to installing and running InVesalius on these systems can be found.
+
+The developer version of InVesalius runs on Windows, macOS, and Linux operating systems.
+On this page, instructions for installing and running InVesalius on these systems can be found.
 
 ## Windows
 
-These instructions are tested on Windows 10 64 bits.
-The packages required by InVesalius can be installed either through a Python system 
-installation or the Anaconda package manager.
-In both cases, the following prerequisites apply:
+These instructions are tested on Windows 10 64-bit.
+The packages required by InVesalius can be installed either through a Python system
+installation, the Anaconda package manager, or using `uv`.
+In all cases, the following prerequisites apply:
 
 ### Prerequisites
+
 - Visual Studio (We are using the 2019 community edition).
   - It's possible to install using [chocolatey](https://chocolatey.org/):
-     ```shell
-      choco install -y visualstudio2019buildtools
-      choco install -y visualstudio2019-workload-vctools
-     ```
+    ```shell
+     choco install -y visualstudio2019buildtools
+     choco install -y visualstudio2019-workload-vctools
+    ```
 - Download InVesalius source code.
-   ```bash
-   git clone --recurse-submodules https://github.com/invesalius/invesalius3
+  ```bash
+  git clone --recurse-submodules https://github.com/invesalius/invesalius3
+  ```
+
+## Installation Options
+
+### Option 1: Using `uv` (Recommended)
+
+`uv` is a modern Python package manager that ensures better dependency management. Follow these steps:
+
+1. Install `uv` on Windows using:
+
+   ```shell
+   scoop install main/uv
```

Comment:
I'd add installing using winget (I think winget it's already included on Window):

```
winget install --id=astral-sh.uv  -e
```